tests/tools/test_dist_part.py

```diff
@@ -0,0 +1,232 @@
+import argparse
```

Comment:
Can we move these test files to the `tests` folder?

Reply:
By these, do you mean both `test_dist_part.py` and `test_launch.py`?

Reply:
which `tests` folder? I think these test files are already under `//tests/`.

Reply:
sorry, I misread.